src/tga/mod.rs

```diff
@@ -6,7 +6,6 @@
 /// A decoder for TGA images
 ///
 /// Currently this decoder does not support 8, 15 and 16 bit color images.
-//TODO add 8, 15, 16 bit color support
```

Comment:
Why was this comment removed? Is it because 8, 15, 16 bits are now supported?

Reply:
it was removed by `rustfmt` apparently because of https://github.com/rust-lang-nursery/rustfmt/issues/2645. I'll move the comment a couple of lines below.

Reply:
Ah, OK.